react/src/base/inputs/SprkCheckbox/SprkCheckbox.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkCheckbox = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    additionalCheckboxClasses,
+    additionalLabelClasses,
+    name,
+    value,
+    disabled,
+    onChangeFunc,
+    id,
+    ariaDescribedBy,
+    ...rest
+  } = props;
+  const internalId = uniqueId('sprk-checkbox-');
+  return (
+    <div
+      className={classnames(
+        'sprk-b-SelectionContainer sprk-b-Checkbox',
+        additionalClasses,
+        {
+          'sprk-b-Checkbox--huge': variant === 'huge',
+        },
+      )}
+      data-analytics={analyticsString}
+      data-id={idString}
+      data-sprk-input="checkbox"
```

Comment:
Can remove the data-sprk-input="checkbox", since its from Vanilla.

Reply:
👍🏽 Fixed!